.github/workflows/i18n-crowdin-upload.yml

```diff
@@ -19,11 +20,14 @@ jobs:
         uses: crowdin/github-action@v1
         with:
           upload_sources: true
+          upload_sources_args: '--dest=public/locales/en-US/grafana.json'
           upload_translations: false
           download_translations: false
           create_pull_request: false
           base_url: 'https://grafana.api.crowdin.com'
           config: 'crowdin.yml'
+          source: 'public/locales/en-US/grafana.json'
+          translation: 'public/locales/%locale%/%original_file_name%'
```

Comment:
Hmm... It is annoying that this is duplicated from the crowdin.yaml file. I wonder maybe if we can get rid of the paths specified in the crowdin.yaml file then? Or, is it still needed for the download script? 

Very annoying!

Reply:
🤔 I'll test this on Monday and come back with feedback then.

Reply:
Removing the whole files section in _crowdin.yml_ leads to an error in the download action:
<img width="669" alt="Screenshot 2024-02-05 at 11 56 50" src="https://github.com/grafana/grafana/assets/48948963/12b890df-b956-4194-9f86-a5c6f5aa7c1f">

Only removing `source` and `translation` from the cobfig leads to another error in the download action:
<img width="671" alt="Screenshot 2024-02-05 at 11 47 53" src="https://github.com/grafana/grafana/assets/48948963/35fb193d-7cac-43fc-80f0-2492f987f4cb">

Furthermore I tested what happens if I remove `source` and `translation` from the config but add both to the download action like I did for the upload and it works. So I tend to implement it that way in order to be consistent with the GH actions (see newest change). 

What do you think?

Reply:
Yup - this seems like the least worst option 😅